src/main/resources/jenkins/plugins/office365connector/Macro/config.jelly

```diff
@@ -0,0 +1,14 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="Template" field="template">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="Value" field="value">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="">
+        <div align="right">
+            <f:repeatableDeleteButton />
+        </div>
+    </f:entry>
+</j:jelly>
```

Comment:
new line is missing at the end of the file

Reply:
Darn missed one 👍